### PR TITLE
Run lintreview through modernize

### PR DESCRIPTION
**Pull request by adrianmoisey**
Relates to https://github.com/markstory/lint-review/pull/149
I'm going to send individual PRs for each part of that PR.

### CODE REVIEW

File: lintreview/config.py [hunk 1]
```diff
@@ -98,7 +100,7 @@ def _update_linter_config(self, linter_config):
         Because linter config is a nested structure, it needs to be
         updated in a somewhat recursive way.
         """
-        for linter, tool_config in linter_config.iteritems():
+        for linter, tool_config in six.iteritems(linter_config):
```
Comment (markstory) [inline lintreview/config.py]: Couldn't you use `.items()` here as well?

File: lintreview/tools/ansible.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```
Comment (markstory) [inline lintreview/tools/ansible.py]: Couldn't we move all the imports to be relative? Python 2.7 supports relative imports.

File: lintreview/tools/commitcheck.py [hunk 3]
```diff
@@ -34,7 +35,7 @@ def execute_commits(self, commits):
         bad = []
         for commit in commits:
             bad.append(self._check_commit(pattern, commit))
-        bad = filter(None, bad)
+        bad = [_f for _f in bad if _f]
```
Comment (markstory) [inline lintreview/tools/commitcheck.py]: Why `_f`? A longer name would be help readability.

Comment (markstory) [inline lintreview/tools/commitcheck.py]: Why `_f`? A longer name would be help readability.

**Comment by adrianmoisey** [comment]
> I think it would also be wise to add py3 builds to travis. Without those it will be very easy to break py3 support and not know.

How do we handle currently failing tests?

**Comment by markstory** [comment]
The Py3 builds can be marked as allow failure until they are passing as a whole.

File: lintreview/tools/commitcheck.py (hunk 3 above)
Comment (adrianmoisey) [inline lintreview/tools/commitcheck.py]: For some reason that's what modernize picked

Comment (adrianmoisey) [inline lintreview/tools/commitcheck.py]: Fixed in 34c0099, not sure if `commit` is the best name for it though.

File: lintreview/tools/ansible.py (hunk 2 above)
Comment (adrianmoisey) [inline lintreview/tools/ansible.py]: I'm not too sure. This change was recommended by modernize. 
Can you give me an example?

**Comment by adrianmoisey** [comment]
Hi @markstory any progress on this?

**Comment by markstory** [comment]
@adrianmoisey I've not had the time to dedicate to this. I'm hoping to get to it next week.

**Pull request merged**